packages/core/admin/server/src/content-types/api-token.ts

```diff
@@ -47,6 +47,13 @@ export default {
       required: true,
       searchable: false,
     },
+    encryptedKey: {
+      type: 'string',
+      minLength: 1,
+      configurable: false,
+      required: true,
```

Comment:
```suggestion
      required: false,
```
Just wondering if we actually need this to be required, since it's not really, and in fact all current tokens won't have it.